### PR TITLE
Surgical scalpels CBM changes

### LINKED ISSUE #39589: Surgical scalpels CBMs cutting quality cant be used to cut up items.
# Describe the bug
The surgical scalpel CBM comes with cutting 1 and fine cutting 5, but when you attempt to cut up something with the butcher menu the tools cutting qualities don't register, they still function as expected for crafting purposes.

# Steps To Reproduce
Spawn a something you can cut up like a sheet, attempt to cut it up while having the bionic installed and no other cutting implements nearby.

# Expected behavior
To be able to use my bionic scalpels to cut stuff.

# Screenshots

# Versions and configuration
- OS: Windows
    - OS Version: 10.0 1903
- Game Version: 0.E-732-gd0f0a96 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food]
]

# Additional context

### PR DESCRIPTION
**Pull request by Pupsi-Mupsi**
#### Summary

 `SUMMARY: None`

#### Purpose of change

This fixes #39589.

- Activate scalpels to cut up things or to use as a weapon if you really need to.
- You need to have your hands free.
- As @Night-Pryanik pointed out to-hit is now +2 instead of -3.

grafik

While active:

grafik

grafik

grafik

Update - add to_hit, piercing and techniques:

grafik

### CODE REVIEW

**Comment by LaVeyanFiend** [comment]
Where's the pierce damage on them 🤔

**Comment by Pupsi-Mupsi** [comment]
hmmm

Go ahead. I'm open for any suggestions.

grafik

**Comment by Pupsi-Mupsi** [comment]
@LaVeyanFiend 
Better?

**Comment by LaVeyanFiend** [comment]
image
(Yes)

File: data/json/items/fake.json [hunk 1]
```diff
@@ -58,7 +58,11 @@
     "copy-from": "fake_item",
     "type": "TOOL",
     "name": { "str": "autonomous surgical scalpels", "str_pl": "autonomous surgical scalpels" },
-    "flags": [ "TRADER_AVOID" ],
+    "description": "A system of surgical grade scalpels.  They allow you to make precise cuts and can also be used as a high-quality butchering tool.",
+    "to_hit": -3,
```
Comment (Night-Pryanik) [inline data/json/items/fake.json:62]: I'm not sure about this. These scalpels are effectively works as continuation of one's arms, so they could be considered as "fists" in terms of to-hit calculation. If so, according to our `game-balance.md`, they get +2 for their "grip".

Comment (Night-Pryanik) [reply data/json/items/fake.json]: No, +2 isn't appropriate either. 
All weapons start with -2 to hit. These scalpels get +2 for having a fist-like "grip", 0 for length, -1 for being "line of damage" weapons, and 0 for neutral balance. Total of -1.

**Pull request merged**